Foundation-Mission-Structure-and-Strategy-Proposal.md

```diff
@@ -0,0 +1,76 @@
+# Node.js Foundation Mission, Structure and Strategy
+
+_Proposal to the Node.js Foundation TSC and Board, by Rod Vagg_
+
+**Goal #1:** Agree on a basic statement of mission for the Foundation, upon which all execution, by each group within the Foundation, can be tested and justified.
+
+**Goal #2:** Agree on a strategy generation and execution structure, and the relationship between the main bodies within that structure, that can best achieve the mission of the Foundation.
+
+## Foundation Mission Statement
+
+**The Node.js Foundation exists to increase the adoption of Node.js.**
+
+This goal is achieved by:
+
+1. Maintaining the health of the core Node.js project
+2. Fostering and encouraging a healthy Node.js open source ecosystem
+3. Fostering and encouraging a healthy commercial ecosystem of training, products and services for Node.js
+
+_This is an executable form of the Node.js Foundation Purpose as presented in its by-laws, Section 1.4_
+
+### Fundamendals
+
+The proposed mission statement stands on a fundamental raison d’être: “increase the adoption of Node.js”
+
+Execution of this mission is divided into three prioritised areas: core, open source ecosystem, commercial ecosystem
+
+Node.js core remains the fundamental focus for the Node.js Foundation, all else builds on this
+
+**The Foundation should operate according to a broad strategy that encompasses the three areas of concern: core, open source ecosystem and commercial ecosystem**
+
```

Comment:
I like that this calls out the 3 areas separately as it helps to identify that we are interested in all 3.